content/yaml/building-a-flutter-app.md

```diff
@@ -16,17 +16,19 @@ Set up local properties
 
     - flutter build apk --release
 
-### Building universal .apk from an existing app bundle with user-specified keys
+### Building universal .apk(s) from existing app bundle(s) with user-specified keys
 
-If your app settings in Codemagic have building Android App Bundles enabled, we will automatically include a script for generating a signed `app-universal.apk` during the YAML export. If you are creating a YAML file from a scratch, add the following script to receive that file:
+If your app settings in Codemagic have building Android App Bundles enabled, we will automatically include a script for generating a signed `app-universal.apk` during the YAML export. If you are creating a YAML file from a scratch, add the following script to receive those file(s):
 
```

Comment:
Change the ending for "... add the following script to receive the .apk file(s):"

Reply:
fixed in fdd4bc9a35859f468413142c28e2c13c8b1f18ff